.code-samples.meilisearch.yaml

```diff
@@ -4,51 +4,51 @@
 # You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
 ---
 get_one_index_1: |-
-  client.getIndex('movies').show()
+  client.index('movies').getRawInfo()
```

Comment:
Oh! can't we use `fetchInfo` instead? We did this in the other SDKs code samples

Reply:
The problem is that it wouldn't be correct with the response provided: 

<img width="784" alt="Screenshot 2020-12-10 at 09 53 47" src="https://user-images.githubusercontent.com/33010418/101749118-a708a380-3acd-11eb-82bb-7a4749951bba.png">

We could ignore it but there is a loss of information between the raw answer and the object.

Reply:
Other solution would be to add `createdAt` and `updatedAt` inside the object parameters like we did with primaryKey. In which case the answer would still be correct.

Reply:
Hum 💩, then we made a mistake with the python PR about the lazy index! 😱 So I made the same on the ruby one... That's not a big deal at the moment, let it as it is!
And that's definitely a good idea to add getters/attributes for `createdAt` and `updatedAt` in the `Index` class!! I will open an issue in integration-guide repo 👍